Мерж с беем.

Drakon341 opened the pull request:
### А чё поменялось-то?
Ну-эээ...

- Харон преределан, а как конкретно сами узнаете.
- Добавлены модные крутящиеся тревоги, которые включаются при коде выше зелёного, пример ниже:

https://user-images.githubusercontent.com/64197080/168336793-d7211ed0-da54-4739-bdfb-1f22532171d6.mp4

- Командные риги получили новые спрайты:

![image](https://user-images.githubusercontent.com/64197080/168337268-80a12a74-8a67-41cc-9bd0-90e279c1b445.png)

- На четвёртой палубе, между комнатой питфиндера и карго появился полноценный переход, теперь не нужно по техам ходить, да...
- Некоторые лоровые планеты поменяли своё название и добавилось несколько новых.
- Админы получили возможность закрывать слоты работок.
- У медалек теперь крутые спрайты(на скрине не все):
-
![image](https://user-images.githubusercontent.com/64197080/168337595-80313e5d-c52e-4ced-9cce-3d10d27c3d40.png)
_Ещё больше обновлений (UPD 17.06.2022)_
-Полностью переработан огнёмёт и связанные с ним системы.
-Полностью переработан блюспейс драйв.
-Добавлен классный эффект, когда в отсеке высокая температура.

https://user-images.githubusercontent.com/64197080/174312076-941ab4fd-bcea-4940-988d-46ee38e9827b.mp4

И это не все изменения и вообще смотрите чейджлог бея, а то мне уже лень перечеслять.

polerno commented:
Я БЛЯ ТЕБЕ ЖМУ РУКУ ЗА ТАКУЮ РАБОТУ БРАТАН

polerno commented:
Блять сломалось всё, ахуеть теперь

polerno commented:
Одно хэмингуэй поспорил что напишит самый короткий и трогательный рассказ. Он написал

**Checks: 5 failed**

Вместе с хэмингуэем плакало половина населения земли

SierraKomodo committed:
Allow any do_after bars to appear over actor or actee

Kolergo commented:
А шо если трейтор кого-то убьёт в начале и весь раунд синий код... Сдохнуть же от эпилепсии можно
А это же морж бэя, за него можно и сдохнуть

Archemagus commented:
Реквестирую анальную пробку для резом с этим эффектом как на уровнях тревоги.

Да пусть он даже она будет синхронизирована с тревогами на Торче

MuckerMayhem committed:
properly fix psychokinesis manifest abilities

SierraKomodo committed:
Use standardized health processing for BSD

Ninetailed committed:
Medbot rework

This is in several parts:

Robot hypo rework

* Robot hypos can now draw from other reagent containers in the bot's inventory.
* Robot hypos get a new nanoui for reagent selection.
* Added bottle rack to crisis and ER bots for use with this.
* Surgeonbot doesn't get a bottlerack. It's expected to be in the OR where it can use bottles with its syringe.
* Added inaprovaline to surgeonbot.
* Added dylovene and dexalin to crisisbot and ERbot.
* Removed tricordrazine from crisisbot and ERbot. No one uses tricordrazine, and you now get dylovene to make it in vivo anyway.

Medbot loadout tweaks

* emergency resposne bot gets an IV bag gripper
* ERbot starts with a roller bed in its rack
* ERbot gets a deployable auto-compressor - not as good as a defib, but necessary since it can't do CPR
* ERbot and crisisbot have their gauze and ointment upgraded to ATKs and ABKs in line with EMT loadout
* Add chem dispenser cartridges to chem gripper

Rename borghydro.dm to borghypo.dm

MuckerMayhem committed:
Merge pull request #32128 from SierraKomodo/hcm-codex

Add mechanics codex to HCMs

MuckerMayhem committed:
Merge pull request #32134 from Ninetailed/surgical_robot_organ_transplant

Fix surgical robots being unable to implant organs

MuckerMayhem committed:
Merge pull request #32135 from Ninetailed/wardrobe_malfunction

Reduce spontaneous wardrobe malfunctions

Bay12Bot committed:
Automatic changelog generation for PR #32128 [ci skip]

MuckerMayhem committed:
Merge pull request #32141 from SierraKomodo/disable-icarus

Disable Icarus crashed vessel spawning

Bay12Bot committed:
Automatic changelog generation for PR #32134 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32135 [ci skip]

MuckerMayhem committed:
Merge pull request #32142 from SierraKomodo/runtime-batch-2022-5-29

Runtime batch 2022-5-29

Bay12Bot committed:
Automatic changelog generation for PR #32141 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32142 [ci skip]

MuckerMayhem committed:
Merge pull request #32143 from SierraKomodo/fix-locker-escapes

Fix being unable to break out of closets

Bay12Bot committed:
Automatic changelog generation for PR #32143 [ci skip]

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32137 from MuckerMayhem/ckeys-assigned-admins

join/leave messages about tickets use admin ckeys

Bay12Bot committed:
Automatic changelog generation for PR #32137 [ci skip]

SierraKomodo committed:
Handle deleted mobs cleanly in ban handlers

- Closes #32145, #32146

SierraKomodo committed:
Fix null target runtimes

- Fixes #32148

SierraKomodo committed:
Fix missing public flag when wrenching AI cores

SierraKomodo committed:
Color-code public and private progress bars

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

dependabot[bot] committed:
Bump actions/cache from 3.0.2 to 3.0.3

Bumps [actions/cache](https://github.com/actions/cache) from 3.0.2 to 3.0.3.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/48af2dc4a9e8278b89d7fa154b955c30c6aaab09...30f413bfed0a2bc738fdfd409e5a9e96b24545fd)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

MuckerMayhem committed:
Merge pull request #32153 from Baystation12/dependabot/github_actions/actions/cache-3.0.3

Bump actions/cache from 3.0.2 to 3.0.3

Bay12Bot committed:
Automatic changelog generation

RareThrumbo committed:
Fix wrong artist linked in music tracks list

🆑 Thrumbo
tweak: Updated the author link on a couple of lobby/jukebox songs to point to the correct artist.
/🆑

"A Drop A Day" is a dude from Italy as noted on his Soundcloud.

"Ghyti" is a post-punk band from Australia and I don't know why they're even linked here.

RareThrumbo committed:
Updated CC license

Spookerton committed:
implemented handling of abstract_type on decls

get_decl returns null for abstract decl types
get_decls and get_decls_unassociated skip nulls
base /decl is abstract

Spookerton committed:
planet sites care about being disallowed

Spookerton committed:
Merge pull request #32149 from SierraKomodo/runtime-batch-2022-05-30

Runtime batch 2022 05 30

Bay12Bot committed:
Automatic changelog generation for PR #32149 [ci skip]

Spookerton committed:
fix bluespace drive eating grabs, robot items, etc

SierraKomodo committed:
Fix missing tape recorder errors #2

- Fixes #32155

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32156 from RareThrumbo/license

Fix wrong artist linked for some songs

MuckerMayhem committed:
Merge pull request #32157 from Spookerton/spkrtn/sys/implement-decl-abstract-handling

implements handling of abstract_type on decls

MuckerMayhem committed:
Merge pull request #32158 from Spookerton/spkrtn/fix/pushy-planets

planet sites care about being disallowed

MuckerMayhem committed:
Merge pull request #32159 from Spookerton/spkrtn/fix/bad-engine-dont-eat-the-robot

fix bluespace drive eating grabs, robot items, etc

Bay12Bot committed:
Automatic changelog generation for PR #32158 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32159 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

SierraKomodo committed:
Block robots from resting

SuckerPunchSyd committed:
code changes to message

Ninetailed committed:
Fix hanging reference left in gripper

Ninetailed committed:
Allow robots to remove tank from shuttle fuel port

MuckerMayhem committed:
Merge pull request #32160 from SierraKomodo/runtime-batch-2022-6-1

Fix missing tape recorder errors #2

MuckerMayhem committed:
Merge pull request #32167 from SierraKomodo/block-robot-resting

Block robots from resting

Bay12Bot committed:
Automatic changelog generation for PR #32167 [ci skip]

SomeAngryMiner committed:
Resprites canister/inflatable boxes/inflatables

SierraKomodo committed:
Fix attacker-less attack log runtimes

- Fixes #30677

`mob_attacker` being `null` is valid - See `/proc/admin_victim_log()`

SierraKomodo committed:
Fix invalid artifact `span_item()` implementations

Fixes #22609

Ensures all implementations return a valid object.
Replaces cases of `new null()` with `..()`.

SierraKomodo committed:
Handle ahelp->irc where target has no mob

Fixes #19240

SierraKomodo committed:
Handle energy nets with no buckled mob

Fixes #19247

SierraKomodo committed:
Fix core field continuing init after qdel

Fixes #32173

nl-208 committed:
adds "set injection for all" button to RUST + style tweaks


Sierra's changes, part 1

- adds a new variable for the clamped form of the user's input that's then applied to all injectors on the network. allows us to only do the math operation once.
- adds some sanity-checking for cancellation or out-of-bounds values, TODO be sure user hasn't walked away from the console/can still reach controls (i.e. isn't incapacitated or lying down)
- shifts the "invalid injection rate" case & message out of the for loop on the global function to prevent chat spam
Sierra's changes, part 2

- sanity checking for if the user's still able to interact with the console (hasn't walked away/gone prone)
- allows the input to properly accept null values (i.e. user cancellation)
Bypasses istype() calls on the for loop

SomeAngryMiner committed:
Update inflatable.dmi

Bay12Bot committed:
Automatic changelog generation

dependabot[bot] committed:
Bump actions/cache from 3.0.3 to 3.0.4

Bumps [actions/cache](https://github.com/actions/cache) from 3.0.3 to 3.0.4.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/30f413bfed0a2bc738fdfd409e5a9e96b24545fd...c3f1317a9e7b1ef106c153ac8c0f00fed3ddbc0d)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

MuckerMayhem committed:
Merge pull request #32178 from Baystation12/dependabot/github_actions/actions/cache-3.0.4

Bump actions/cache from 3.0.3 to 3.0.4

MuckerMayhem committed:
Merge pull request #32132 from SierraKomodo/bsd-health

Use standardized health processing for BSD

Bay12Bot committed:
Automatic changelog generation

dependabot[bot] committed:
Bump actions/setup-python from 3.1.2 to 4

Bumps [actions/setup-python](https://github.com/actions/setup-python) from 3.1.2 to 4.
- [Release notes](https://github.com/actions/setup-python/releases)
- [Commits](https://github.com/actions/setup-python/compare/98f2ad02fd48d057ee3b4d4f66525b231c3e52b6...d09bd5e6005b175076f227b13d9730d56e9dcfcb)

---
updated-dependencies:
- dependency-name: actions/setup-python
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

MuckerMayhem committed:
Merge pull request #32174 from SomeAngryMiner/engineeringp8

Resprites canister/inflatable boxes/inflatables

MuckerMayhem committed:
Merge pull request #32175 from SierraKomodo/runtime-batch-2022-06-05

Runtime batch 2022 06 05

MuckerMayhem committed:
Merge pull request #32180 from Baystation12/dependabot/github_actions/actions/setup-python-4

Bump actions/setup-python from 3.1.2 to 4

Bay12Bot committed:
Automatic changelog generation for PR #32174 [ci skip]

SierraKomodo committed:
Merge pull request #32080 from MuckerMayhem/actually-fix-psychokinesis

Properly fix psychokinesis manifest abilities

Bay12Bot committed:
Automatic changelog generation for PR #32080 [ci skip]

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32150 from SierraKomodo/progress-bars-over-actor-or-actee

Allow any do_after bars to appear over actor or actee

Bay12Bot committed:
Automatic changelog generation for PR #32150 [ci skip]

MuckerMayhem committed:
Merge pull request #32165 from nl-208/rust-injection-console-tweaks-ii

adds "set injection for all" button to RUST + style tweaks

MuckerMayhem committed:
Merge pull request #32170 from SuckerPunchSyd/dev

adds gold slime reaction

Bay12Bot committed:
Automatic changelog generation for PR #32165 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32170 [ci skip]

Drakon341 committed:
возвращаем..

Drakon341 committed:
брррр

Bay12Bot committed:
Automatic changelog generation

nl-208 committed:
fixes bug with non-appearing stamina bar

stamina bars now update using the priv_prog_bar icon state instead of the now-nonexistent prog_bar

SierraKomodo committed:
Merge pull request #32181 from nl-208/stamina-bar-fix

fixes bug with non-appearing stamina bar

Bay12Bot committed:
Automatic changelog generation for PR #32181 [ci skip]

Bay12Bot committed:
Automatic changelog generation

nl-208 committed:
Removes old overriding logic for bot damage

Removes an overriding updatehealth() proc specific to bots that hasn't worked in years and prevented them from actually registering incoming damage in its current state.

Bay12Bot committed:
Automatic changelog generation

CrimsonShrike committed:
Merge pull request #32171 from CrimsonShrike/FireAtTheDisco

* Turf fires

* Converts old ZAS fire to hotspots for differentiation

* Reworks flamethrowers

* Simple animals now will burn and accumulate fire stacks

* Exosuit flamethrowers

Bay12Bot committed:
Automatic changelog generation for PR #32171 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

Imienny1 committed:
Minor Raiders Change

Spookerton committed:
Merge pull request #32185 from nl-208/no-unkillable-bots

Fixes AI-controlled bots being unkillable

Bay12Bot committed:
Automatic changelog generation for PR #32185 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Spookerton committed:
Merge pull request #32162 from Ninetailed/med_module_improvements

Robot medical module improvements

Bay12Bot committed:
Automatic changelog generation for PR #32162 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Drakon341 committed:
Merge branch 'dev' into merge

Drakon341 committed:
Merge https://github.com/Baystation12/Baystation12 into merge

Drakon341 committed:
чиним

SierraKomodo committed:
Block entering mechs while buckled

- Fixes #32176

Ninetailed committed:
Update NanoUI maps

MuckerMayhem committed:
Merge pull request #32197 from Ninetailed/update_nanoui_maps

Update nanoui maps

Bay12Bot committed:
Automatic changelog generation for PR #32197 [ci skip]

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32172 from Ninetailed/robot_shuttle_fuel_tank

Robots can now remove shuttle H2 tanks from ports

MuckerMayhem committed:
Merge pull request #32192 from Imienny1/MinorRaidersChange

Minor survivability update for raiders

Bay12Bot committed:
Automatic changelog generation for PR #32172 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32192 [ci skip]

Bay12Bot committed:
Automatic changelog generation

SierraKomodo committed:
Fix missing death handler for rubble

SierraKomodo committed:
Fix ethereal jaunting off the map errors

- Fixes #32191

SierraKomodo committed:
Fix spawning money with preset amounts and no user

- Fixes #32187

CrimsonShrike committed:
Fix skybox and openspace light issues


Fixes lighting issues with zmimic

SierraKomodo committed:
Handle infinite charge weapons in mech HUD overlay

- Fixes #32188

SierraKomodo committed:
Fix charging failure message order

- Fixes #31140

SierraKomodo committed:
Fix handling of empty speak lists in holopads

- Fixes #30793

SierraKomodo committed:
Fix closed uplink runtimes when locking consoles

- Fixes #32036

SierraKomodo committed:
Fixes broken bone handling with no internals

- Fixes #32193

External organs with no internal organs seems perfectly valid - Legs, arms, etc.

SierraKomodo committed:
Fix empty mob objectives in cryopod flushing

- Fixes #32194

SierraKomodo committed:
Fix empty stack machinery runtimes

- Fixes #31188

Empty stacks are valid in the case of borgs with no cable or other material charge left.

SierraKomodo committed:
Fix continued loop after nulling affecting

- Fixes #29536

MuckerMayhem committed:
Merge pull request #32200 from CrimsonShrike/FixSkybox

Fix skybox and openspace light issues

MuckerMayhem committed:
add access datum for science-only storage

MuckerMayhem committed:
fix event ship being multi z

MuckerMayhem committed:
fix wormhole proc spawning WAY too many wormholes

MuckerMayhem committed:
make charon and gup faster

Bay12Bot committed:
Automatic changelog generation

Spookerton committed:
observers can respawn freely during roundstart

SierraKomodo committed:
Add gas meter to charon main fuel line

MuckerMayhem committed:
Merge pull request #32208 from Spookerton/spkrtn/cng/salmon-permissivity

observers can respawn freely during roundstart

MuckerMayhem committed:
Merge pull request #32198 from SierraKomodo/fix-rubble-health

Fix missing death handler for rubble

Bay12Bot committed:
Automatic changelog generation for PR #32208 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32198 [ci skip]

Ryan180602 committed:
add EC science duty gloves + science bowman

also gives bowman headsets antenna in N-S sprites
a

Bay12Bot committed:
Automatic changelog generation

SierraKomodo committed:
Add vv handlers for standardized health vars

Spookerton committed:
Merge pull request #32201 from SierraKomodo/runtime-batch-2022-06-17

Runtime batch 2022-06-19

Bay12Bot committed:
Automatic changelog generation for PR #32201 [ci skip]

Spookerton committed:
Merge pull request #32205 from MuckerMayhem/event-ship-fix

Fix event ship being multi z

Spookerton committed:
Merge pull request #32209 from SierraKomodo/charon-fuel-meter

Add gas meter to charon main fuel line

Spookerton committed:
Merge pull request #32206 from MuckerMayhem/charon-gup-tweaks

Tweaks Charon and GUP to make them faster

Bay12Bot committed:
Automatic changelog generation for PR #32209 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32206 [ci skip]

Spookerton committed:
Merge pull request #32204 from MuckerMayhem/wormhole-tweaks

Fix wormhole proc spawning WAY too many wormholes

Spookerton committed:
Merge pull request #32203 from MuckerMayhem/science-locker-access

Add access type for science storage

Bay12Bot committed:
Automatic changelog generation for PR #32203 [ci skip]

SierraKomodo committed:
Transfer IV drip detach logic to a proc

- Fixes #28653, #26226
- Updates feedback and visible messages

Spookerton committed:
re-add /datum to config, update minimum versions

Spookerton committed:
add isdatum, clean up is_drone & isPlunger

Spookerton committed:
un-FQ controller vars

Bay12Bot committed:
Automatic changelog generation

Spookerton committed:
remove irrelevant tmp & static sugars

static confers tmpness
const confers staticness

Imienny1 committed:
Make Dufflebags Great Again

MuckerMayhem committed:
Merge pull request #32199 from Ryan180602/sciessential

Add EC science duty gloves + Science bowman + Bowman sprite changes

Bay12Bot committed:
Automatic changelog generation for PR #32199 [ci skip]

MuckerMayhem committed:
Merge pull request #32211 from SierraKomodo/runtime-batch-2022-06-21

Runtime batch 2022-06-21

Bay12Bot committed:
Automatic changelog generation for PR #32211 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Spookerton committed:
remove leftover uses of regex_* macros

Spookerton committed:
remove leftover uses of list_find macro

Spookerton committed:
remove tests related to regex_* & list_find macros

SierraKomodo committed:
Check computer before using it

- fixes #25880

SierraKomodo committed:
Nullify mech bodyparts after update_icon calls instead of before

- Fixes #31314

SierraKomodo committed:
Fix node checks when updating clamps

- Fixes #30691

Adds the same conditions used in `/close()`

SierraKomodo committed:
Remove unnecessary proc/var calls in canister icon updates

SierraKomodo committed:
Adjust canister pressure indicator sprite levels

MuckerMayhem committed:
add an admin log when players gain psionics

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32218 from SierraKomodo/runtime-batch-2022-06-22

Runtime batch 2022 06 22

MuckerMayhem committed:
fix simple mobs being unable to force doors

SierraKomodo committed:
Fix invalid web hitby() override

- Fixes #31847

SierraKomodo committed:
Fix mutations list emptiness check in slimes

- Fixes #31843

SierraKomodo committed:
Merge pull request #32216 from Spookerton/spkrtn/cng/lingering-evils

Spkrtn/cng/lingering evils

SierraKomodo committed:
Merge pull request #32212 from Spookerton/spkrtn/cng/assorted0621

Spkrtn/cng/assorted0621

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32220 from MuckerMayhem/simple-mob-door-force-fix

Fix simple mobs being unable to force doors

MuckerMayhem committed:
Merge pull request #32215 from MuckerMayhem/psi-log

Add an admin log when players gain psionics

Bay12Bot committed:
Automatic changelog generation for PR #32220 [ci skip]

Ryan180602 committed:
add compass from nebula


a


sdad

Bay12Bot committed:
Automatic changelog generation

SierraKomodo committed:
Merge pull request #32213 from Imienny1/MakeDufflebagsGreatAgain

Make dufflebags worth using

Bay12Bot committed:
Automatic changelog generation for PR #32213 [ci skip]

SierraKomodo committed:
Merge pull request #32196 from Ryan180602/compas

Port Compass from Nebula (> Polaris)

Bay12Bot committed:
Automatic changelog generation for PR #32196 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

Spookerton committed:
update play_ambience; areas cache vent_pumps

Ryan180602 committed:
add flobsters

Ryan180602 committed:
lets coordinates passively show in inv

Spookerton committed:
isEquipped can check a slot, uses it for duffles

dependabot[bot] committed:
Bump JamesIves/github-pages-deploy-action from 4.3.3 to 4.3.4

Bumps [JamesIves/github-pages-deploy-action](https://github.com/JamesIves/github-pages-deploy-action) from 4.3.3 to 4.3.4.
- [Release notes](https://github.com/JamesIves/github-pages-deploy-action/releases)
- [Commits](https://github.com/JamesIves/github-pages-deploy-action/compare/8817a56e5bfec6e2b08345c81f4d422db53a2cdc...e6d003d0839927f5a4b998bfd92ed8e448fde37a)

---
updated-dependencies:
- dependency-name: JamesIves/github-pages-deploy-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump styfle/cancel-workflow-action from 0.9.1 to 0.10.0

Bumps [styfle/cancel-workflow-action](https://github.com/styfle/cancel-workflow-action) from 0.9.1 to 0.10.0.
- [Release notes](https://github.com/styfle/cancel-workflow-action/releases)
- [Commits](https://github.com/styfle/cancel-workflow-action/compare/a40b8845c0683271d9f53dfcb887a7e181d3918b...bb6001c4ea612bf59c3abfc4756fbceee4f870c7)

---
updated-dependencies:
- dependency-name: styfle/cancel-workflow-action
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

MuckerMayhem committed:
Merge pull request #32210 from SierraKomodo/standardized-health-vv-handlers

Add vv handlers for standardized health vars

MuckerMayhem committed:
Merge pull request #32219 from SierraKomodo/canister-pressure-indicator-lights

Canister pressure indicator lights

MuckerMayhem committed:
Merge pull request #32221 from SierraKomodo/runtime-batch-2022-06-23

Runtime batch 2022 06 23

Bay12Bot committed:
Automatic changelog generation for PR #32210 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #32219 [ci skip]

MuckerMayhem committed:
Merge pull request #32226 from Ryan180602/gpscoord

Lets GPS coordinates show while in Inventory

Bay12Bot committed:
Automatic changelog generation for PR #32226 [ci skip]

MuckerMayhem committed:
Merge pull request #32227 from Ryan180602/flobster

Adds Foam Flip Flobsters

MuckerMayhem committed:
Merge pull request #32229 from Baystation12/dependabot/github_actions/JamesIves/github-pages-deploy-action-4.3.4

Bump JamesIves/github-pages-deploy-action from 4.3.3 to 4.3.4

Bay12Bot committed:
Automatic changelog generation for PR #32227 [ci skip]

MuckerMayhem committed:
Merge pull request #32230 from Baystation12/dependabot/github_actions/styfle/cancel-workflow-action-0.10.0

Bump styfle/cancel-workflow-action from 0.9.1 to 0.10.0

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

Drakon341 committed:
Merge https://github.com/Baystation12/Baystation12 into merge

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

sneedgamer847195 commented on code/modules/culture_descriptor/location/locations_human.dm:
Даже не думайте мержить культуры и локейшены с бэя, бля.
Они удалили кучу классных планет и заменили их скучным бутлеговым мусором, чего только стоит выпил Лордании и замена Кингстона буквально той же планетой, но холодной.

Evie7056 replied:
Думали сразу отревертить их в последствии, лмао

MuckerMayhem committed:
Merge pull request #32228 from Spookerton/spkrtn/cng/isEquipped

isEquipped can check a slot, uses it for duffles

Bay12Bot committed:
Automatic changelog generation

MuckerMayhem committed:
Merge pull request #32225 from Spookerton/spkrtn/cng/ambience

play_ambience is probably no longer area size dependent for performance

SierraKomodo committed:
Give pilots access to post-flight reports

MuckerMayhem committed:
Merge pull request #32236 from SierraKomodo/pilot-post-flight-report

Give pilots access to post-flight reports

Bay12Bot committed:
Automatic changelog generation for PR #32236 [ci skip]

Bay12Bot committed:
Automatic changelog generation

Bay12Bot committed:
Automatic changelog generation

Drakon341 committed:
Merge https://github.com/Baystation12/Baystation12 into merge
just necessary

Drakon341 committed:
Merge remote-tracking branch 'upstream/dev' into merge

Evie7056 committed:
Фикс униформ и макрос для ИИ

Evie7056 committed:
Фикс команд тгс

Drakon341 committed:
сдвиг по фазе у факела

Drakon341 committed:
ой зона

Drakon341 committed:
абракадабра

Drakon341 committed:
ИИИИХААААА

Drakon341 committed:
на пути к капитализму

Drakon341 committed:
на пути к капитализму намбер ту

Evie7056 committed:
Красный дракон блять съел дме.

Drakon341 committed:
ыыы

Drakon341 committed:
земля плоская

Evie7056 committed:
э

Drakon341 committed:
я своего рода математик

Drakon341 committed:
Merge branch 'merge' of https://github.com/Drakon341/Baystation12RU into merge

Drakon341 committed:
цезарь переходит рубикон

Drakon341 committed:
скажем нет розовым прогресс барам

Drakon341 committed:
странные канистры

Drakon341 committed:
Revert "странные канистры"

This reverts commit a55619837bd7f30eaf5c3b60913d3373a196f7b7.

Drakon341 committed:
чиним гы

Drakon341 committed:
нет рюкзакам

Evie7056 committed:
ммм, хуйня.

Evie7056 committed:
Пускаем лохов на 513 на сервер, лол.

Evie7056 committed:
дубль два.

Drakon341 committed:
чиним знаки и куртки

Evie7056 committed:
Блять тикер.

Evie7056 committed:
Merge remote-tracking branch 'drakosha/merge' into merge

Evie7056 committed:
Дракон не глупый, он просто совершает ошибки

Drakon341 committed:
наполеон переходил границу
